### ISSUE TITLE
[Bug]  Start-ADTMsiProcess Uninstall will not work for product that are not installed on system.

### ISSUE THREAD

**Issue by bulax** [PSAppDeployToolkit/PSAppDeployToolkit#1105]
### Prerequisites

- [X] Ensure you write a short, descriptive title after [Bug] above.
- [X] Make sure to [search for any existing issues](https://github.com/psappdeploytoolkit/psappdeploytoolkit/issues) before filing a new one.
- [X] Verify you are able to reproduce the issue with the [latest released version](https://www.github.com/psappdeploytoolkit/psappdeploytoolkit/releases/latest)

### PSAppDeployToolkit version

4.0.2

### Describe the bug

Running command Start-ADTMsiProcess -Action UnInstall -ProductCode '{039694F1-2108-4B3E-8575-85C245210F94}' on device where MSI is not installed will throw error.

### Steps to reproduce

1. Create wrapper with install/uninstall any MSI
2. Try to run UnInstall on device where MSI is not installed
![image](https://github.com/user-attachments/assets/a03b30f4-04a7-4666-a15f-fdee811ce73e)

### Environment data

```console
OsName               : Microsoft Windows 10 Enterprise
OsOperatingSystemSKU : EnterpriseEdition
OsArchitecture       : 64-bit
WindowsVersion       : 2009
WindowsProductName   : Windows 10 Enterprise
WindowsBuildLabEx    : 19041.1.amd64fre.vb_release.191206-1406
OsLanguage           : en-US
OsMuiLanguages       : {en-US}
KeyboardLayout       : en-US
TimeZone             : (UTC+00:00) Dublin, Edinburgh, Lisbon, London
HyperVisorPresent    : True
CsPartOfDomain       : False
CsPCSystemType       : Desktop
```

**Comment by mjr4077au** [comment]
This has been addressed in https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/pull/1110. Once this PR is merged into our development branch, and that branch is put into a release, the issue will be resolved.

**Comment by mjr4077au** [comment]
Closing this one off as the committed fix will be in the next point release.
